ckanext/dcatapchharvest/tests/fixtures/dataset-media-types.xml

```diff
@@ -14,5 +14,17 @@
         <dcterms:format rdf:datatype="http://www.w3.org/2001/XMLSchema#string">application/json</dcterms:format>
       </dcat:Distribution>
     </dcat:distribution>
+    <dcat:distribution>
+      <dcat:Distribution>
+        <dcat:mediaType rdf:resource="http://www.iana.org/assignments/text/plain"/>
+        <dcterms:format rdf:resource="http://publications.europa.eu/resource/authority/file-type/GRID_ASCII"/>
+      </dcat:Distribution>
+    </dcat:distribution>
+    <dcat:distribution>
+      <dcat:Distribution>
+        <dcat:mediaType rdf:datatype="http://www.w3.org/2001/XMLSchema#string">text/plain</dcat:mediaType>
```

Comment:
maybe we could use some other media type in the test,
e.g., `text/calendar`  https://www.iana.org/assignments/media-types/text/calendar

Reply:
I can add a test for that that too.